FTP/FTP/Server.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MyFTP
+{
+    /// <summary>
+    /// Server class
+    /// </summary>
+    public class Server
+    {
+        private TcpListener listener;
+        private CancellationTokenSource token = new CancellationTokenSource();
```

Comment:
```suggestion
        private CancellationTokenSource token = new();
```
Ещё один пережиток сурового прошлого C#